app/views/users/_edit.haml

```diff
@@ -229,7 +233,20 @@
             = link_to t(".request_export_photos"), export_photos_user_path, method: :post,
               class: "btn btn-default"
 
-      .col-md-6
+    .row
+      .col-md-12
+        %h3
+          = t(".import.headline", accountname: current_user.diaspora_handle)
+        %p
+          = t(".import.body")
+          %strong
+            = t(".import.warning")
+        .form-group
+          .btn.btn-primary{id: "import_account", data: {toggle: "modal", target: "#importAccountModal"}}
```

Comment:
I think after validating the modal and while the import is ongoing, this blue button "import another account" should be removed and replaced by a sentence "Account xxx@ccc is being imported, please come back in a few minutes" like we are doing for the export.

Reply:
Button is now invisible for users with a pending / ongoing import.